plugins/lang/js/dataset_js.cc

```diff
@@ -26,7 +26,8 @@ v8::Handle<v8::Object>
 DatasetJS::
 create(std::shared_ptr<Dataset> dataset, JsPluginContext * context)
 {
-    auto obj = context->Dataset->GetFunction()->NewInstance();
+    v8::Isolate* isolate = v8::Isolate::GetCurrent();
```

Comment:
Since I don't know the V8 interface, I thought that `EscapableHandleScope` was some kind of guard to free allocated resources.  It is now clear that it is not!